File: sources/nim/09_fib_recursive/main.nim
# task 09 fib_recursive — expected output: 102334155
# build: nim c -d:release -o:prog main.nim    run: ./prog

proc fib(n: int64): int64 =
  if n < 2: n
  else: fib(n - 1) + fib(n - 2)

echo fib(40)

File: sources/julia/09_fib_recursive/main.jl
# task 09 fib_recursive — expected output: 102334155
# build: julia main.jl    run: julia main.jl

function fib(n::Int64)::Int64
    if n < 2
        return n
    end
    return fib(n - 1) + fib(n - 2)
end

function main()
    println(fib(Int64(40)))
end

main()
